terraform/example/bot.tf.example

```diff
@@ -0,0 +1,35 @@
+# Teleport Machine ID Bot creation example
+
+locals {
+  bot_name = "example"
+}
+
+resource "random_password" "bot_token" {
+  length           = 32
+  special          = false
+}
+
+resource "time_offset" "bot_example_token_expiry" {
+  offset_hours = 1
+}
+
+resource "teleport_provision_token" "bot_example" {
```

Comment:
I think I definitely agree with separating the creation of the token and bot for Terraform.